README.md

```diff
@@ -1,30 +1,30 @@
-# Elasticsearch::Drain
+# Elasticsearch Drain
 
-TODO: Write a gem description
+The purpose of this utility is to drain documents from Elasticsearch nodes in an AutoScaling Group.
 
-## Installation
+This will help you do Elasticsearch node replacement while keeping the cluster healthy.
 
-Add this line to your application's Gemfile:
+Consider the following deployment procedure:
```

Comment:
Might want to include a note that this is a useful way to do an upgrade if you can't update binaries in place, like when you bake versioned AMIs.